test/core/MakerOperations.t.sol

```diff
@@ -1050,4 +1047,11 @@ contract MakerOperationsTest is MangroveTest, IMaker {
     (, local) = mgv.config(olKey);
     assertEq(local.tickPosInLeaf(), 2);
   }
+
+  function test_leaf_update_both_first_and_last() public {
+    uint ofr0 = mgv.newOfferByVolume(olKey, 0.01 ether, 1 ether, 1000000, 0);
```

Comment:
I think we should avoid adding more ByVolume in tests. I find they are more difficult to reason about because the price (and thus tick) isn't apparent.

Reply:
fixed in 962779950c4407e50d1a2c9882e0f1ff6bfb4d56